src/systems/eu/implementations/react/components/description-list/src/DescriptionListWithData.jsx

```diff
@@ -32,10 +32,10 @@ const DescriptionListWithData = ({ items, className, ...props }) => {
         }
 
         return (
-          <Fragment key={item.term}>
+          <div key={item.term}>
```

Comment:
There are still a few extra div to be converted to Fragment, here

Reply:
Sorry, updated these as well